.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.6'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Where did we obtain those credentials?

Reply:
You will have to setup your account on pypi, and github let's you store those as secrets.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets